mobile_verifier/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+mod cell_type;
+mod error;
+mod mobile;
+mod reward_share;
+mod reward_speed_share;
+
+pub mod cli;
+pub mod server;
+pub mod subnetwork_rewards;
+
+pub use error::{Error, Result};
+pub use server::Server;
+
+use poc_store::FileStore;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
+use std::io;
+
+pub async fn write_json<T: ?Sized + serde::Serialize>(
```

Comment:
I do _not_ think pushing json to buckets is the right thing to do here. We're not going to support this json format publicly. We need proto messages to operate on instead of this approach

Reply:
where are we even using this?  I don't see it being called.

Reply:
it's called a bunch of times in subnetwork_rewards afaict

Reply:
nevermind me, my language server was apparently very tired this morning and couldn't find the references

Reply:
Fixed